nsxt/resource_nsxt_policy_service.go

```diff
@@ -504,24 +491,11 @@ func resourceNsxtPolicyServiceRead(d *schema.ResourceData, m interface{}) error
 	}
 
 	var obj model.Service
-	if isPolicyGlobalManager(m) {
-		client := gm_infra.NewServicesClient(connector)
-		gmObj, err := client.Get(id)
-		if err != nil {
-			return handleReadError(d, "Service", id, err)
-		}
-		lmObj, err := convertModelBindingType(gmObj, gm_model.ServiceBindingType(), model.ServiceBindingType())
-		if err != nil {
-			return err
-		}
-		obj = lmObj.(model.Service)
-	} else {
-		client := infra.NewServicesClient(connector)
-		var err error
-		obj, err = client.Get(id)
-		if err != nil {
-			return handleReadError(d, "Service", id, err)
-		}
+	client := infra.NewServicesClient(getSessionContext(d, m), connector)
+	var err error
+	obj, err = client.Get(id)
```

Comment:
We can save two lines of code here and in similar places by not defining `obj` and `err` and using `:=` on this line